helix-core/src/main/java/org/apache/helix/model/ClusterConfig.java

```diff
@@ -109,7 +109,11 @@
     // https://github.com/apache/helix/wiki/Weight-aware-Globally-Evenly-distributed-Rebalancer#rebalance-coordinator
     //
     // Default to be true.
-    GLOBAL_REBALANCE_ASYNC_MODE
+    GLOBAL_REBALANCE_ASYNC_MODE,
+
+    // The target size of task thread pools for each participant. This is the global value
+    // that's used when participants don't specify their individual pool sizes.
+    GLOBAL_TARGET_TASK_THREAD_POOL_SIZE
```

Comment:
Can you share the reason why we change to GLOBAL_XXXX instead of DEFAULT_XXXX? I think default value is closer to how it is used. If participant specifies their values individually, it is not globally applied, right?

Reply:
IMO, there could be 2 potential misunderstandings if we use GLOBAL.
1. This size is the total thread count of all the participant thread pool size.
2. This size will be applied to every single participant.

To avoid those confusing, using DEFAULT is easier.

Regarding the internal DEFAULT value which is 40,
1. it is internal only, so even we name the above one as DEFAULT, it won't confuse the users.
2. I think we only have NOT_SET which is -1. There is no DEFAULT TARGET thread pool size, right? The 40 is the default thread pool size which is corresponding to the current thread poll size.

Reply:
As a counter argument, "global target" sounds like the target value that's applied globally, which is correct here. The confusion over "default" may also apply to developers too. 

Personally, I think different people have different interpretation to the field names, but what I can do is to provide well-written javadocs that explain clearly what these fields do on CRUD APIs, then the naming doesn't matter as much. What do you think?

Reply:
Ultimately, you can make the call. I still think global makes more sense in this context and more intuitive for devs to understand, but it seems that Jiajun has used "default" for the WAGED rebalancer related configs that serve a similar purpose.

I don't think it'd be hugely off either way, so you could either get a third opinion or make the call yourself :) But I agree that we should make this clear in both JavaDoc and a user manual of some sort.

Reply:
Based on a vote within the dev team, we have decided on using "GLOBAL_XXX".